utils/prof_utils.py

```diff
@@ -619,6 +619,10 @@ def est_sn_from_prof(prof_data, period, alpha=3.):
     scattered: boolean
         When true, the profile is highly scattered
     """
+    # Check profile is normalised
+    if max(prof_data) > 1.:
```

Comment:
can this condition be true without it being normalised?

Reply:
There's a small chance that it will if there's a float rounding error (max will be 1.0000001) but then the profile will barely be changed

Reply:
Can't an un-normalised profile have a maximum less than 1?

Reply:
O now I see what you're saying. It could but it is extremely unlikely as the raw profile is on the order of 10^7. I could remove this check and always normalise it to be safe?

Reply:
yeah it might just be best to always normalise since it's such a small computational expense anyway

Reply:
Changed